extension/extesionhelper/factory.go

```diff
@@ -0,0 +1,101 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package extensionhelper
```

Comment:
The package name is correct here but the folder name (and thus all imports) has a typo

Reply:
Also my favourite comment that I don't think you want to do 😄 : I think this should be moved up to live in the `extension` folder directly so people can use `extension.NewFactory(...)`

Reply:
followup PR